model/db/heart_disease_bycounty_ddl.sql

```diff
@@ -0,0 +1,55 @@
+CREATE TABLE "State_symbol" (
+	"StateName"	TEXT,
+	"Symbol"	TEXT,
+	PRIMARY KEY("Symbol")
+);
+
+CREATE TABLE "Heart_Disease_Mortality_County" (
+	"State"	TEXT,
```

Comment:
Abbr

Reply:
I am planning to change it to Code rather than Abbr